apps/web/utils/supabase/server.ts

```
@@ -1,11 +1,98 @@
+                single: async () => ({ data: {}, error: null }),
+              }),
+            }),
+            update: () => ({
+              eq: () => ({
+                select: () => ({
+                  single: async () => ({ data: {}, error: null }),
+                }),
+              }),
+            }),
+            delete: () => ({
+              eq: () => ({ then: async () => ({}) }),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)